src/base/preferences.cpp

```diff
@@ -1056,45 +1045,45 @@ bool Preferences::isMagnetLinkAssocSet()
 
 void Preferences::setTorrentFileAssoc(const bool set)
 {
-    QSettings settings("HKEY_CURRENT_USER\\Software\\Classes", QSettings::NativeFormat);
+    QSettings settings(u"HKEY_CURRENT_USER\\Software\\Classes"_qs, QSettings::NativeFormat);
 
     // .Torrent association
     if (set)
     {
-        const QString oldProgId = settings.value(".torrent/Default").toString();
-        if (!oldProgId.isEmpty() && (oldProgId != "qBittorrent"))
-            settings.setValue(".torrent/OpenWithProgids/" + oldProgId, "");
-        settings.setValue(".torrent/Default", "qBittorrent");
+        const QString oldProgId = settings.value(u".torrent/Default"_qs).toString();
+        if (!oldProgId.isEmpty() && (oldProgId != QLatin1String("qBittorrent")))
+            settings.setValue((u".torrent/OpenWithProgids/"_qs + oldProgId), QString());
```

Comment:
```suggestion
            settings.setValue((u".torrent/OpenWithProgids/" + oldProgId), QString());
```

Reply:
Same here.

Reply:
Debagger points me to some template in `QStringBuilder` class.

Reply:
All such overloads exist in order to avoid unnecessary construction of `QString` class instance, which improves performance. Even if you use a new `_qs` literal that avoids copying data, some redundancy is still present.

Reply:
OK, all done.